### DISCUSSION TITLE
Go to type definition of hinted type (infered)

### DISCUSSION THREAD

**Discussion by snowmead** [helix-editor/helix#8283] [Q&A] [answered]
In this example, the rust analyzer can infer the type `ApplicationCommandInteraction`. I would like to go to the definition of this type but I cannot select it in any way to execute `g+d`.

I tried the workspace symbol picker `space+S`, but it does not pick up symbols from imported dependencies. 

Is there a method that I am not aware of?

![image](https://github.com/helix-editor/helix/assets/94772640/2664d73a-58b7-4a45-8993-4d81aa83136b)

**Comment by pascalkuthe** [comment, marked as answer]
you can use `gy` im most instances

**Reply by snowmead** [reply]
Ah thank you!
